analysers/experiments/cpu.py

```diff
@@ -178,22 +156,20 @@ def sortAndGet(field, asc):
 #####################################################
 
 
-CassandraRDD = sc.cassandraTable(cassandraKeyspace, "trial_cpu_core")
+CassandraRDD = sc.cassandraTable(cassandraKeyspace, "trial_cpu_core") \
+    .select("cpu_min", "cpu_max", "cpu_q1", "cpu_q2", "cpu_q3", "cpu_p95", "cpu_median", "cpu_num_data_points", "cpu_mean", "cpu_me", "trial_id") \
+    .where("experiment_id=? AND container_id=?", experimentID, containerID)
 CassandraRDD.cache()
 
-nOfCores = CassandraRDD.select("cpu_mean") \
-        .where("experiment_id=? AND container_id=?", experimentID, containerID) \
-        .first()                                                                                           
-                                                                                                           
-nOfCores = len(nOfCores["cpu_mean"]) 
+nOfCores = len(CassandraRDD.first()["cpu_mean"])
```

Comment:
Please open an issue pointing to this line, because we need to retrieve this information from the container information, once stored in the database.